recipes/stopwordsiso/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "stopwordsiso" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5be2b4f25afc296fca4f626710b98168949057f81a655e2dbdf670ddfce2af50
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - setuptools
+
+test:
+  imports:
+    - stopwordsiso
+  commands:
+    - pip check
+  requires:
+    - pip
+  commands:
```

Comment:
The checks now seem to be here twice. Can you please clean up this part, then this should be ready to go!

Reply:
arg, sorry, not sure what happened here. fixed it.

Reply:
Thanks! Maybe a GitHub error. Looks good - will merge this when the tests pass. 

Thanks for your contribution!